src/components/AnnotationTarget/__tests__/AnnotationTarget-test.tsx

```diff
@@ -63,11 +66,28 @@ describe('AnnotationTarget', () => {
         test('should pass the required props to the underlying anchor', () => {
             const wrapper = getWrapper({ className: 'ba-Test' });
 
-            expect(wrapper.props()).toMatchObject({
+            // Get the inner anchor element
+            const innerAnchor = wrapper.childAt(0);
+            expect(innerAnchor.props()).toMatchObject({
                 className: 'ba-AnnotationTarget ba-Test',
                 role: 'button',
                 tabIndex: 0,
             });
         });
     });
+
+    describe('scroll if isActive', () => {
+        afterEach(() => {
+            jest.resetAllMocks();
```

Comment:
I don't think this is needed. It should be called by default. If not, we need to update our jest config.